MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -356,8 +358,63 @@ MeshLib::Mesh* VtkMeshConverter::convertUnstructuredGrid(vtkUnstructuredGrid* gr
 		elements[i] = elem;
 	}
 
-	return new MeshLib::Mesh(mesh_name, nodes, elements);
+	MeshLib::Mesh* mesh = new MeshLib::Mesh(mesh_name, nodes, elements);
+	convertScalarArrays(*grid, *mesh);
+
+	return mesh;
+}
+
+void VtkMeshConverter::convertScalarArrays(vtkUnstructuredGrid &grid, MeshLib::Mesh &mesh)
+{
+	vtkPointData* point_data = grid.GetPointData();
+	if (point_data == nullptr)
```

Comment:
i guess it's possible that VTK data include cell values without any point values